neo4j/resultsummary.go

```diff
@@ -218,8 +218,9 @@ type Notification interface {
 	// Not all notifications have a unique position to point to and in that case the position would be set to nil.
 	Position() InputPosition
 	// Severity returns the severity level of this notification.
-	// Deprecated: Severity will be removed in 6.0.
 	// Please rely on SeverityLevel (or RawSeverityLevel) instead.
+	//
+	// Deprecated: Severity will be removed in 6.0.
```

Comment:
That's the wrong way around, isn't it? Shouldn't the "pleas use XYZ" be part of the deprecation message?

Reply:
fixed in f6df686a5c69a0375f26b2521d7ff2e18bae67b2